src/pfe/file-watcher/server/test/functional-test/configs/project.config.ts

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
```

Comment:
Are there any specific difference between `project.config.ts` and `app.config.ts`? If they are all config files for project, we can merge them into one config file only for project.

Reply:
Yeah. `project.config.ts` is project specific configs. `app.config.ts` is just the test app configuration.

Reply:
I still can't see the obvious difference between the two files, but if you want make it two files I fine with it, but leave detailed comments or instructions to ensure people can add project config codes to the correct file in the future.

Reply:
Yes, that is a good idea. I added the comments on top of the file.